lib/src/component/component_manager.dart

```diff
@@ -42,6 +42,6 @@ class ComponentManager {
   }
 
   ComponentPool<T> getComponentPool<T extends Component<V>, V>() {
-    return _componentPool[T];
+    return _componentPool[T] as ComponentPool<T>;
```

Comment:
`!`?

Reply:
No, in this case we are doing a casting because the `_componentPool` is typed on `ComponetPool<dynamic>`